### PR TITLE
Feature/mini contact

### PR DESCRIPTION
**Pull request by ambtlv**

### CODE REVIEW

File: cwel/component/mini-contact/mini-contact.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{% macro default(title, title2, text) %}
+    <section class="container--dark">
+        <div class="container">
+            <div class="mini-contact">
+                <div class="mini-contact-grid">
```
Comment (dstuessy) [inline cwel/component/mini-contact/mini-contact.html]: @ambtlv I would call this class `mini-contact__col` if these are supposed to be different parts of the mini-contact

File: cwel/component/mini-contact/mini-contact.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                    <h4 class="mini-contact__title">{[ title ]}</h4>
+                    <p class="mini-contact__text">
+                        {[ text ]}
+                    </p>
+                </div>
+                <div class="mini-contact-grid">
+                    <h4 class="mini-contact__title">{[ title2 ]}</h4>
+                    <ul class="mini-contact__text">
```
Comment (dstuessy) [inline cwel/component/mini-contact/mini-contact.html]: @ambtlv I'm thinking the contents of `.mini-contact-grid` would be CMSable. Might want to omit the class names for elements inside.

File: cwel/component/mini-contact/mini-contact.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                        <li>Monday to Friday 8am - 8pm</li>
+                        <li>Saturday 9am - 5pm</li>
+                        <li>Sunday 10am - 4pm</li>
+                    </ul>
+                </div>
+                <div class="mini-contact-grid mini-contact-grid--alignleft">
+                    <button class="mini-contact__button mini-contact__button--bright"></button>
```
Comment (dstuessy) [inline cwel/component/mini-contact/mini-contact.html]: @ambtlv Seeing as nathan's work with buttons is in develop, you could try use the standard `.button` classes!

File: cwel/component/mini-contact/mini-contact.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    </section>
+{% endmacro %}
+
+{% macro dualcolumn(title, title2, text) %}
+    <section class="container--dark">
+        <div class="container">
+            <div class="mini-contact--dual-col">
+                <div class="mini-contact-grid--half">
+                    <h4 class="mini-contact__title--centered">{[ title ]}</h4>
+                    <p class="mini-contact__text--centered">
```
Comment (dstuessy) [inline cwel/component/mini-contact/mini-contact.html]: @ambtlv I would put the `--centered` modifier on the `mini-contact-grid` element

File: cwel/style/component/_mini-contact.scss [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+        margin-bottom: $s_md;
+        color: $white;
+
+        @include breakpoint('lg') {
+            text-align: center;
+        }
+    }
+}
+
+ul {
```
Comment (dstuessy) [inline cwel/style/component/_mini-contact.scss]: @ambtlv I would get rid of this generic style as these should have been handled by the reset stylesheet. I would just change the lists in mini contact to `p` tags.

**Review by dstuessy — CHANGES_REQUESTED** [review]
Good stuff, man. Just a few pointers to address. Next step is the galen tests! 😄

File: sandbox/component/mini-contact/style.scss [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+.mini-contact__button {
+    width: 100%;
+    min-height: 5.5rem;
+    margin-bottom: $s_md;
+    color: $white;
+    text-transform: uppercase;
+    border: 0;
+    border-radius: 4rem;
+
+    &--small { // Modifiers for testing - actual sizes to be created
```
Comment (Decad) [inline sandbox/component/mini-contact/style.scss]: @ambtlv What does this mean? When do the actual sizes get created?

Comment (ambtlv) [reply sandbox/component/mini-contact/style.scss]: @Decad - this is just testing and part of the actual component coding. The real buttons created by Nathan will essentially go here. I was just trying to match the UX design...just fillers at this stage.

**Comment by ambtlv** [comment]
@dstuessy ...this is now ready for another review based off of previous review/comments. Cheers.

File: cwel/component/mini-contact/mini-contact.html [hunk 7]
```diff
@@ -52,13 +60,15 @@ <h4 class="mini-contact__title--centered">{[ title2 ]}</h4>
     <section class="container--dark">
         <div class="container">
             <div class="mini-contact--dual-col">
-                <div class="mini-contact-grid--half">
-                    <h4 class="mini-contact__title--centered">{[ title ]}</h4>
-                    <p class="mini-contact__text--centered">
-                        {[ text ]}
-                    </p>
+                <div class="mini-contact__col--half mini-contact__col--centered">
```
Comment (dstuessy) [inline cwel/component/mini-contact/mini-contact.html]: @ambtlv `mini-contact__col mini-contact__col--half mini-contact__col--centered`

**Review by dstuessy — CHANGES_REQUESTED** [review]

**Comment by ambtlv** [comment]
@dstuessy ...further updates. Please review.

File: cwel/component/mini-contact/mini-contact.html [hunk 8]
```diff
@@ -57,31 +62,36 @@ <h4 class="mini-contact__title">
...
 {% endmacro %}
 
 {% macro singlecolumn(title, text) %}
-    <section class="container--dark">
-        <div class="container">
-            <div class="mini-contact--single-col">
-                <div class="mini-contact__col--full mini-contact__col--centered">
+    <section class="mini-contact">
+        <div class="mini-contact__container">
+            <div class="mini-contact__content--single-col">
```
Comment (dstuessy) [inline cwel/component/mini-contact/mini-contact.html]: `mini-contact__content mini-contact__content--single-col`

File: cwel/component/mini-contact/mini-contact.html [hunk 9]
```diff
@@ -95,16 +105,18 @@ <h4 class="mini-contact__title">
 {% endmacro %}
 
 {% macro singlecolumnmix(title) %}
-    <section class="container--dark">
-        <div class="container">
-            <div class="mini-contact--single-col">
-                <div class="mini-contact__col--full mini-contact__col--centered">
+    <section class="mini-contact">
+        <div class="mini-contact__container">
+            <div class="mini-contact__content--single-col">
```
Comment (dstuessy) [inline cwel/component/mini-contact/mini-contact.html]: mini-contact__content mini-contact__content--single-col

File: cwel/style/component/_mini-contact.scss [hunk 10]
```diff
@@ -1,5 +1,13 @@
 // Container related markup
 .mini-contact {
+    background: $c_primary;
```
Comment (dstuessy) [inline cwel/style/component/_mini-contact.scss]: `$c_miniContact_bg: $c_primary !default;`

`background: $c_miniCrontact_bg;`

https://robots.thoughtbot.com/sass-default

**Review by dstuessy — CHANGES_REQUESTED** [review]

**Comment by ambtlv** [comment]
@dstuessy ...further updates :-)

**Review by dstuessy — APPROVED** [review]

**Review by Decad — APPROVED** [review]

**Comment by dstuessy** [comment]
@ambtlv ready to merge and write the docs, man! Good job!

**Pull request merged**